forge-gui/res/cardsfolder/b/baru_fist_of_krosa.txt

```diff
@@ -3,7 +3,7 @@ ManaCost:3 G G
 Types:Legendary Creature Human Druid
 PT:4/4
 T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Forest | TriggerZones$ Battlefield | Execute$ TrigPump | TriggerDescription$ Whenever a Forest enters the battlefield, green creatures you control get +1/+1 and gain trample until end of turn.
-A:AB$ Token | Cost$ Discard<1/Card.namedBaru, Fist of Krosa> | TokenAmount$ 1 | TokenScript$ g_x_x_wurm | TokenOwner$ You | TokenPower$ X | TokenToughness$ X | CostDesc$ Grandeur — Discard another card named CARDNAME: | SpellDescription$ Create an X/X green Wurm creature token, where X is the number of lands you control.
+A:AB$ Token | Cost$ Discard<1/Card.namedBaru, Fist of Krosa> | TokenAmount$ 1 | TokenScript$ g_x_x_wurm | TokenOwner$ You | TokenPower$ X | TokenToughness$ X | CostDesc$ Grandeur — Discard another card named Baru, Fist of Krosa: | SpellDescription$ Create an X/X green Wurm creature token, where X is the number of lands you control.
```

Comment:
Don't know when we'll get back to this again so might be worth adding
`DeckHints:Ability$Token & Type$Wurm`